PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/Invisicam.rbxmx

```diff
@@ -165,6 +164,21 @@ end
 
 -- Update. Called every frame after the camera movement step
 function Invisicam:Update()
+	-- Make sure we still have a Torso
```

Comment:
Could probably do it when the Torso's ancestry changes. I did this just to make it bulletproof; it only checks if it doesn't have a reference, after all.